src/main/java/com/revlearn/team1/service/TransactionService.java

```diff
@@ -0,0 +1,9 @@
+package com.revlearn.team1.service;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public interface TransactionService
```

Comment:
nice job with the interface! that's gonna make testing hella easier!

Reply:
This is Daniel's work.  I pulled his branch into mine.

Reply:
TAKE THE DAMN CREDIT